giga_connectome/utils.py

```diff
@@ -196,6 +200,7 @@ def check_path(path: Path, verbose=True):
         ]
         n = str(max(existing_numbers) + 1) if existing_numbers else "1"
         path = path_parent / f"{path.stem}_{n}{ext}"
-        if verbose:
-            print(f"Specified path already exists, using {path} instead.")
+
+        gc_log.debug(f"Specified path already exists, using {path} instead.")
```

Comment:
not sure if this one should be info or debug?

Reply:
This is related to #76....
I am inclined to call this info for what we have for now.
But after #76 this should not be relevant?